deploy/hyperlane-kms-address.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Compute Hyperlane validator KMS address
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  vars_files:
+    - group_vars/hyperlane/vault.yml
+
+  vars:
+    hyperlane_agents_network: "{{ hyperlane_agents_network | default('') }}"
+    hyperlane_validator_index: "{{ hyperlane_validator_index | default('') }}"
+
+  tasks:
+    - name: Validate required inputs
+      assert:
+        that:
+          - hyperlane_agents_network | length > 0
+          - hyperlane_validator_index | length > 0
+          - validator is defined
+          - validator[hyperlane_agents_network] is defined
+          - validator[hyperlane_agents_network][hyperlane_validator_index] is defined
+        fail_msg: >-
+          Missing inputs or vault keys. Expected:
+          validator[{{ hyperlane_agents_network }}][{{ hyperlane_validator_index }}]
```

Comment:
You could add those input validation for `hyperlane.yml` as `pre_tasks` so you don't need to repeat them later.